deltachat-ffi/src/lib.rs

```diff
@@ -3135,10 +3172,11 @@ impl<T: Default, E: std::fmt::Display> ResultExt<T> for Result<T, E> {
         }
     }
 
-    fn log_err(&self, context: &context::Context, message: &str) {
-        if let Err(err) = self {
-            error!(context, "{}: {}", message, err);
-        }
+    fn log_err(self, context: &context::Context, message: &str) -> Result<T, E> {
```

Comment:
Maybe it is more clear to just add a line returning `self`, without changing the rest of the code.

Reply:
Hmm, I guess that's arguable.  My brain parses .map_err() more easily then an if let expression.